definitions/npm/@entur/sdk_v1.x.x/flow_v0.25.x-/sdk_v1.x.x.js

```diff
@@ -0,0 +1,1032 @@
+type $entur$sdk$TransportMode =
+    | 'air'
+    | 'bus'
+    | 'cableway'
+    | 'coach'
+    | 'funicular'
+    | 'lift'
+    | 'metro'
+    | 'rail'
+    | 'tram'
+    | 'unknown'
+    | 'water'
+
+// All valid values for the "mode" parameter to JourneyPlanner
+ type $entur$sdk$QueryMode =
+    | 'air'
+    | 'bicycle'
+    | 'bus'
+    | 'cableway'
+    | 'car'
+    | 'car_dropoff'
+    | 'car_park'
+    | 'car_pickup'
+    | 'coach'
+    | 'foot'
+    | 'funicular'
+    | 'lift'
+    | 'metro'
+    | 'rail'
+    | 'tram'
+    | 'transit'
+    | 'water'
+
+type $entur$sdk$LegMode =
+    | $entur$sdk$TransportMode
+    | 'bicycle'
+    | 'car'
+    | 'foot'
+
+type $entur$sdk$TransportSubmode =
+    | 'localBus'
+    | 'regionalBus'
+    | 'expressBus'
+    | 'nightBus'
+    | 'sightseeingBus'
+    | 'shuttleBus'
+    | 'schoolBus'
+    | 'railReplacementBus'
+    | 'airportLinkBus'
+    | 'internationalCoach'
+    | 'nationalCoach'
+    | 'localTram'
+    | 'cityTram'
+    | 'metro'
+    | 'local'
+    | 'regionalRail'
+    | 'interregionalRail'
+    | 'longDistance'
+    | 'international'
+    | 'touristRailway'
+    | 'nightRail'
+    | 'airportLinkRail'
+    | 'internationalCarFerry'
+    | 'nationalCarFerry'
+    | 'regionalCarFerry'
+    | 'localCarFerry'
+    | 'internationalPassengerFerry'
+    | 'localPassengerFerry'
+    | 'sightseeingService'
+    | 'highSpeedVehicleService'
+    | 'highSpeedPassengerService'
+    | 'internationalFlight'
+    | 'domesticFlight'
+    | 'helicopterService'
+    | 'telecabin'
+    | 'funicular'
+
+
+type $entur$sdk$DestinationDisplay = {
+    frontText: string,
+    ...
+}
+
+type $entur$sdk$Config = {|
+    clientName: string,
+    hosts?: {|
+        journeyPlanner?: string,
+        geocoder?: string,
+        nsr?: string,
+        scooters?: string,
+    |},
+    headers?: {|
+        [string]: string
+    |},
+    fetch?: (
+        url: string,
+        init?: Object,
```

Comment:
`Object` is deprecated and now an alias to `any`, could it be type more precisely?

Reply:
In this specific case, I would prefer to keep the type not too specific. Users of this library can substitute the default `fetch` implementation with their own, for advanced use cases. I know that `node-fetch` uses some really specific union types for this `init` parameter, while the browser `fetch` uses some other type. If I restrict this `init` parameter too much, I think that can lead to type conflicts for users that are not really helpful.

There are some other usages of Object in this file though, I'll see if I can improve those.

Reply:
Then please replace them with `any`, at least it's explicit.